.github/workflows/publish-commit-bottles.yml

```diff
@@ -0,0 +1,60 @@
+name: Publish and commit bottles
+
+on: repository_dispatch
+
+jobs:
+  upload:
+    runs-on: ubuntu-latest
```

Comment:
Maybe we can run this on macOS? What ya think?

Reply:
The Jenkins Bottle workflow runs on Ubuntu (the Jenkins master). It could use hosted runners but... I trust the Homebrew/brew Docker image more :)

Reply:
I would rather run anything on Linux that we can as the workers are generally faster to boot/run.